ios/MaasCore/Sources/MaasComponents/Components/BottomSheet/Alert/AlertBottomSheet.swift

```diff
@@ -0,0 +1,185 @@
+import SwiftUI
+
+public struct AlertBottomSheet: View {
+
+    public class ViewModel: ObservableObject {
+
+        @Published public var image: UIImage?
+        @Published public var title: String?
+        @Published public var subtitle: String?
+        @Published public var buttons: [AlertBottomSheet.Button]
+
+        public init(image: UIImage?, title: String?, subtitle: String?, buttons: [AlertBottomSheet.Button]) {
+            self.image = image
+            self.title = title
+            self.subtitle = subtitle
+            self.buttons = buttons
+        }
+    }
+
+    @EnvironmentObject var viewModel: ViewModel
+
+    @Binding var isPresenting: Bool
+
+    @Themeable(AlertBottomSheetConstants.init) var constants
+
+    public init(isPresenting: Binding<Bool>) {
+        self._isPresenting = isPresenting
+    }
+
+    public var body: some View {
+        VStack(spacing: constants.contentVerticalSpacing) {
+
+            if let title = viewModel.title {
+                Text(title)
+                    .textStyle(constants.titleTextStyle)
+            }
+
+            if let subtitle = viewModel.subtitle {
+                Text(subtitle)
+                    .textStyle(constants.subtitleTextStyle)
+            }
+
+            ForEach(viewModel.buttons.indices, id: \.self) {
+                viewModel.buttons[$0].view { isPresenting = false }
+            }
+        }
+        .multilineTextAlignment(.center)
+    }
+}
+
+// MARK: - Button
+
+// TODO: Delete Me
```

Comment:
May be more helpful for future us to write why 🙂
```suggestion
// TODO: Use standard Button styles instead
```